src/services/providers/helpers/couchdbHelper.js

```diff
@@ -21,8 +21,8 @@ const request = async (token, options = {}) => {
         url: utils.resolveUrl(baseUrl, '../_session'),
         withCredentials: true,
         body: {
-          name,
-          password,
+          name: name || '',
+          password: password || '',
```

Comment:
We may not need `password: password || ''`, only push `name:''` can show credentials page.